interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: git is the local on your own computer that holds the files and folders. Git also has commands in terminal to push or create a branch from the repository that was copied from Github. Github is a used in your browser to let the files and folders from each local push it to expand on that code. Changes to that code can be pushed to merge into one repository. 
 
-Researched answer:
+Researched answer: Git is used for sharing and working on code and allows for many people to add to that certain code. Gitthub is a cloud based host that allows you to manage git repositories.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: True or False as it makes a decision based off signs such as greater sign > or less than sign <
 
-Researched answer:
+Researched answer: Boolean represents one of two values: true or false. Example: 7 > 5 would output true
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: The index is the the position in that array. It starts with an index of zero and so on. The difference between Index and value is the actual string or numbers inside the array is considered a value.  [1,2,3,4] the index of 1 would be zero. The values consist of the numbers 1,2,3,4 inside that array.
 
-Researched answer:
+Researched answer: The Indexes are continous numbers that are like a label for every element in the array. The content of each element is called the value. 
 
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer: An array is a set of data in a list consisting of a zero index. A string is one of the data types used in JavaScript. An Array holds strings or numbers in brackets, a string will consist of names or numbers that is listed with quotation marks. The two go hand in hand when you list strings in an array.
 
-Researched answer:
+Researched answer:  Strings are similar to arrays with a few differences. Usually, the array size is set, while strings can have a fluid number of elements. Arrays can contain any data type  while strings are usually characters. 
```

Comment:
We can modify arrays and influence their size as well. What about built-in methods? Would that be a similarity or difference between them?